apps/web/src/modules/job/components/JobCard/styled.ts

```diff
@@ -4,6 +4,7 @@ export const CardContainer = styled('div')`
   display: flex;
   flex-direction: column;
   align-items: flex-start;
+  margin-bottom: 10vh;
```

Comment:
Could you please explain why use `margin-bottom: 10vh;`

Reply:
this issue https://github.com/2110336-2565-2/sec2-group9-modela/pull/20#issuecomment-1416735588 + magic number 😂